username/username_traviscidetails.py

```diff
@@ -2,12 +2,12 @@
 
 # Credits: https://github.com/int0x80/tcispy
 
-# The required token (github_travis_key) can be generated from the link https://github.com/settings/tokens/new by following the requirement present at https://travispy.readthedocs.io/en/stable/getting_started/
+# The required token (github_access_token) can be generated from the link https://github.com/settings/tokens/new by following the requirement present at https://travispy.readthedocs.io/en/stable/getting_started/
 
 import base
-import config as cfg
 import sys
 from termcolor import colored
+from config import github_access_token
```

Comment:
Code like this results in issues outlined in https://github.com/DataSploit/datasploit/issues/176

Please have a look and if that suggestion makes sense please configure accordingly.

Reply:
While I don't disagree with your statement (if something can't be imported, it will cause that error) - it was present before my series of patches, just in another form, see below.

```
➜  datasploit git:(master) ✗ python username/username_traviscidetails.py test

[+] Checking Travis-CI user (author and associated committer) details

Traceback (most recent call last):
  File "datasploit.py", line 55, in <module>
    main(user_input, options.output)
  File "datasploit.py", line 46, in main
    usernameOsint.run(user_input, output)
  File "/opt/dsploit-testing/usernameOsint.py", line 9, in run
    osint_runner.run("username", "username", username, output)
  File "/opt/dsploit-testing/osint_runner.py", line 29, in run
    data = x.main(m_input)
  File "/opt/dsploit-testing/username/username_traviscidetails.py", line 37, in main
    token = TravisPy.github_auth(cfg.github_travis_key)
AttributeError: 'module' object has no attribute 'github_travis_key'
➜  dsploit-testing git:(master) 
```

So now, between your previous commit (187b726) and the check in this PR
(`if github_access_token != "XYZ" and github_access_token != "":`) if there is no token, it will just skip the module. 

```
➜  datasploit git:(traviscidetails) ✗ python username/username_traviscidetails.py test

[+] Checking Travis-CI user (author and associated committer) details

[!] Error: No github token for Travis CI found. Skipping
```

Additionally the github_access_token has been used in datasploit since the first commit (c03da0f) , and the setup instructions have always directed the user to `mv config_sample.py config.py` to prevent this, so I find the likelihood of this specific key being missing (or being considered a _new key_ like in the referenced issue) very small. However, I agree that we should still account for the edge cases regardless. I pushed this commit with the import and sanity check in the style that @upgoingstar has been pushing to this repo recently. But maybe he has some guidance, or input by others, on this that we can begin to deploy across all the modules to create/maintain more consistency and help us guide other contributors coding styles.

Reply:
I agree its a problem in the entire framework. And that's the reason i raised an issue for the whole framework. also pointed out here that this is what i have observed. And exactly as you suggested i am trying to get that code guidelined in the system so that the templates have these samples but lets see when @upgoingstar gets to doing that.